SPECS/python3/CVE-2023-45803.patch

```diff
@@ -0,0 +1,92 @@
+From 397b16e6c86d79439a75a1d50babb8ed08373f14 Mon Sep 17 00:00:00 2001
```

Comment:
For each of these patches, please practice polite open-source-community-etiquette by updating the headers to reflect the original author for the upstream.

add “.patch” to the end of the github patch url to recover.